src/components/logs/LinePart.tsx

```diff
@@ -0,0 +1,22 @@
+import { ParsedSpan } from 'ansicolor';
+import { ListItemText, styled } from '@mui/material';
+
+interface Props {
+    parsedLine: ParsedSpan;
+    lastPart?: boolean;
+}
+
+function LinePart({ parsedLine, lastPart }: Props) {
+    const StyledLogLinePart = styled('span')(parsedLine.css);
```

Comment:
The `ansicolor` package handles styling things based on how is parses stuff. So just using that straight up. We can customize this in the future if we want to.